packages/components/organisms/f-header/src/components/Header.vue

```diff
@@ -62,6 +63,11 @@ export default {
             default: ''
         },
 
+        customNavLinks: {
```

Comment:
Should there be any validation for the values passed in via the array? (the main thing I'm thinking would be, what happens if someone specifies `text` but not `url`, or vice versa?).

Reply:
This is possible and I'll look into this on Monday, hopefully an issue that easy to test wouldn't make it out of local development 😅

Reply:
I'd like to think so too, but always worth considering what happens if people misuse the contract that's provided – kind of acts as a form of documentation for anyone using the prop in the wrong way then.